### PR TITLE
Fix #3599 Can no longer save text format and editor configuration form, and upgrade bootstrap_utilities from 2.0.0 to 3.0.0

### LINKED ISSUE #3599: Can no longer save the Standard text formatter and editor configuration form
## Problem/Motivation

Since Drupal 10.3 (QS `2.11.x` and `main`) there is an error when saving the config form for text formats in Quickstart.

When saving the form I see the following error.
```
The website encountered an unexpected error. Try again later.

Symfony\Component\Validator\Exception\LogicException: The LangcodeRequiredIfTranslatableValues constraint can only operate on the root object being validated. in Drupal\Core\Config\Plugin\Validation\Constraint\LangcodeRequiredIfTranslatableValuesConstraintValidator->validate() (line 26 of core/lib/Drupal/Core/Config/Plugin/Validation/Constraint/LangcodeRequiredIfTranslatableValuesConstraintValidator.php).
```
When I try to save the Basic HTML  formatter/editor i see this error
```
'table_remove_width_height' is an unknown key because filters.bootstrap_utilities_table_filter.id is bootstrap_utilities_table_filter (see config schema type filter_settings.*).
```
image

### Describe the bug
I can't save the standard formatter.

### To Reproduce
1. Go to /admin/config/content/formats
2. Using the operations links, click configure for one of the mentioned text formats
3. Scroll to the bottom of the form and click save.
4. See errors.

## Proposed resolution
Add patches to fix the schema for these plugins.
- bootstrap_utilities
- ckeditor_indentblock

### Expected behavior
I should be allowed to edit my text formats and editors.

### Additional context
See: https://www.drupal.org/project/bootstrap_utilities/issues/3301133
See: https://www.drupal.org/project/ckeditor_indentblock/issues/3457094
See: https://www.drupal.org/project/drupal/issues/3449851

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Since Drupal 10.3 (QS `2.11.x` and `main`) there is an error when saving the config form for text formats in Quickstart.

This should allow you to save at least the standard and basic html text formatter/editors.
I've added 2 patches

## Related issues
Fixes #3599 

Drupal issues
https://drupal.org/i/3301133
https://drupal.org/i/3457094
https://drupal.org/i/3402570

## How to test
1. Go to /admin/config/content/formats
2. Using the operations links, click configure for one of the mentioned text formats
3. Scroll to the bottom of the form and click save.
4. Open a page edit form and verify you can indent and save tables.

Testing the source mode:
1. Create new or edit existing content using the CKEDitor source function.
2. Add a paragraph `<p class="bg-red">something</p>`
3. Toggle the Source button to see if bg-red is stripped.
4. Try indenting the `<p>`
5. Toggle the Source button to see if indenting is stripped.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [x] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change requires release notes.

### CODE REVIEW

**Review by bberndt-uaz — APPROVED** [review]
I was able to save the Basic text format successfully, but when I try to save the Standard text format, I get this error message:

image
<p> </p>

We are currently using a patch from [this Drupal core issue](https://www.drupal.org/project/drupal/issues/3410100), which we added in this Quickstart PR: [Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags](https://github.com/az-digital/az_quickstart/pull/3114). Perhaps we should replace the patch with [the merge request](https://www.drupal.org/project/drupal/issues/3410100#comment-15707744) @joeparsons created recently?

**Comment by joeparsons** [comment]
> I was able to save the Basic text format successfully, but when I try to save the Standard text format, I get this error message:
> 
> image
> We are currently using a patch from [this Drupal core issue](https://www.drupal.org/project/drupal/issues/3410100), which we added in this Quickstart PR: [Closes #3113 Standard text format cannot be edited in the UI without removing allowed tags](https://github.com/az-digital/az_quickstart/pull/3114). Perhaps we should replace the patch with [the merge request](https://www.drupal.org/project/drupal/issues/3410100#comment-15707744) @joeparsons created recently?

The MR on that issue is an incomplete fix IMHO.  The patch we've been using just disables/removes the problematic constraint entirely.  Not sure why that patch isn't working here (?)

**Review by bberndt-uaz — DISMISSED** [review]
With the new patch, I was able to save the Standard text format configuration successfully.

image

**Review by joeparsons — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -148,9 +148,16 @@
...
+            "drupal/bootstrap_utilities": {
+                "Fix schema (3301133)": "https://www.drupal.org/files/issues/2024-08-02/bootstrap_utilities-3301133-12_0.patch"
+            },
             "drupal/cas": {
                 "ServiceNotFoundException when logging out (2948185)": "https://www.drupal.org/files/issues/cas_logout_error-2948185-2.patch"
             },
+            "drupal/ckeditor_indentblock": {
+                "Symfony\\Component\\Validator\\Exception\\LogicException: The LangcodeRequiredIfTranslatableValues constraint can only operate on the root object being validated. (3457094)": "https://www.drupal.org/files/issues/2024-07-25/ckeditor_indentblock-3457094-schema-error.patch",
+                "Plugin blocks other classes on P elements (3402570)": "https://www.drupal.org/files/issues/2024-05-15/ckeditor_indentblock-class-names-3402570-6.patch"
+            },
```
Comment (trackleft) [inline composer.json:160]:
```suggestion
```

File: composer.json [hunk 2]
```diff
@@ -148,9 +148,16 @@
                 "Fix schema (3217958)": "https://gist.githubusercontent.com/joeparsons/7d36cefca83823e93b44cb5b36374b04/raw/c495d02fddcb8364f9d6163bcd74f75e3d47f24f/3217958-5-x.patch",
                 "Subtheme not D10 Compatible (3291051)": "https://www.drupal.org/files/issues/2023-10-27/3291051-5.1.x-subtheme-not-d10-compatible.patch"
             },
+            "drupal/bootstrap_utilities": {
+                "Fix schema (3301133)": "https://www.drupal.org/files/issues/2024-08-02/bootstrap_utilities-3301133-12_0.patch"
+            },
```
Comment (trackleft) [inline composer.json:153]:
```suggestion
```

**Review by bberndt-uaz — APPROVED** [review]
Tested in Probo again and was able to save the Standard text format configuration successfully. In a text paragraph, I was able to set the alignment and indentation for some text, which results in classes being added to the `<p>` element. (Applying text styles from the Styles dropdown creates a `<span>` element with classes instead.)

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**